### PR TITLE
Hotfix/unit test fdescribe

### PR DESCRIPTION
**Pull request by ammarhaiderbjss**
## Description
- Uses lodash's `cloneDeep` when working with `initialState` in the vehicle checks reducer tests.
- The nested array structure of the state object and using the spread operator (`...initialState`) to copy state was causing cross contamination issues when running tests.

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image

### CODE REVIEW

**Review by TomBillingtonUK — APPROVED** [review]
Nice find :)

**Review by MarkPatt89 — APPROVED** [review]
Awesome work

File: src/pages/test-report/cat-be/components/vehicle-checks/vehicle-checks.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.test-report-vehicle-checks {
+vehicle-checks {
```
Comment (szabikr) [inline src/pages/test-report/cat-be/components/vehicle-checks/vehicle-checks.scss:1]: Should we name this component `vehicle-checks-office`? The selector is called that, but maybe we could name the files that way as well

**Review by edwardwilson — CHANGES_REQUESTED** [review]
I would like to investigate the reducer further before we merge as I think there might be a run time bug here

**Review by edwardwilson — APPROVED** [review]
I have checked and the behaviour seen in the unit tests does not appear at runtime!

**Pull request merged**